.github/workflows/wheel_manylinux.yml

```diff
@@ -0,0 +1,66 @@
+name: wheel - manylinux2014
+
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  wheel_manylinux:
+    name: manylinux2014
+    runs-on: ${{ matrix.os }}
+    container:
+      image: quay.io/pypa/manylinux2014_x86_64:latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: [cp37-cp37m, cp38-cp38]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        run: |
+          echo "::add-path::/opt/python/${{ matrix.python-version }}/bin/"
+      - name: check C++ compiler and Python
+        run: |
+          which gcc
+          gcc --version
+          g++ --version
+          python --version
+          which python
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade --no-cache-dir pip
+          python -m pip install --no-cache-dir "poetry<2,>=1.0"
+          python -m pip install --no-cache-dir "mypy-protobuf~=1.0"
+      - name: Build libraries
+        run: |
+          make
+      - name: Build wheel
+        run: |
+          python -m poetry build -v
+      - name: Fix wheel
+        env:
+          PLAT: manylinux2014_x86_64
+        run: |
+          for whl in dist/*.whl; do
```

Comment:
Is the for loop necessary?

Reply:
actually, no (I didn't know we could pip install with wildcards)

Reply:
Oh, this is not the install line. I don't know if auditwheel can take wildcards

Reply:
It does. On the other hand, this loop also removes the old wheel.

Should we move fixed wheel to another location or just keep the current workflow?

Reply:
`multibuild`'s approach is interesting. It puts the repaired wheel in a new directory, avoiding the need to delete the old one. The new directory is called `wheelhouse`.

Reply:
Your approach is ok. I don't like the for loop.

Maybe just output the wheel into the base directory (i.e., not dist). That would work, right? And it would eliminate a few lines of code.

Reply:
I put the fixed wheel to a local folder called `wheelhouse`, and removed the loop.

(`/wheelhouse` is not a writable location with macOS image)